quirks/change-password-URLs.json

```diff
@@ -13,6 +14,7 @@
     "myaccount.ea.com": "https://myaccount.ea.com/cp-ui/security/index",
     "news.ycombinator.com": "https://news.ycombinator.com/changepw",
     "nintendo.com": "https://accounts.nintendo.com/password/edit",
+    "santander.de" : "https://meine.santander.de/login/#/passwort-zuruecksetzen",
```

Comment:
Same question here.

Reply:
It's a forgot password link, where you can request a new password.